lib/matplotlib/tests/test_ticker.py

```diff
@@ -63,6 +72,31 @@ def test_LogLocator():
     test_value = np.array([0.5, 1., 2., 4., 8., 16., 32., 64., 128., 256.])
     assert_almost_equal(loc.tick_values(1, 100), test_value)
 
+    # test label locator
```

Comment:
This should have a ```@cleanup``` decorator to make sure that the plot is closed correctly after the test